classes/class-wc-connect-nux.php

```diff
@@ -546,6 +545,44 @@ public function show_banner_after_connection() {
 			) );
 		}
 
+		public function show_tos_banner() {
+			if ( ! $this->should_display_nux_notice_for_current_store_locale() ) {
+				return;
+			}
+
+			if ( ! $this->should_display_nux_notice_on_screen( get_current_screen() ) ) {
+				return;
+			}
+
+			if ( isset( $_GET['wcs-nux-tos'] ) && 'accept' === $_GET['wcs-nux-tos'] ) {
+				WC_Connect_Options::update_option( 'tos_accepted', true );
+
+				$this->tracks->opted_in( 'tos_banner' );
+
+				wp_safe_redirect( remove_query_arg( 'wcs-nux-tos' ) );
+				exit;
+			}
+
+			$country = WC()->countries->get_base_country();
+			/* translators: %s: list of features, potentially comma separated */
+			$description_base = __( "WooCommerce Services is almost ready to go! Once you connect your store you'll have access to %s.", 'woocommerce-services' );
+			$feature_list     = $this->get_feature_list_for_country( $country );
+
+			$this->show_nux_banner( array(
+				'title'          => __( 'Connect your store to activate WooCommerce Services', 'woocommerce-services' ),
+				'description'    => esc_html( sprintf( $description_base, $feature_list ) ),
+				'button_text'    => __( 'Connect', 'woocommerce-services' ),
```

Comment:
The verbiage here in title, description, and button_text still evokes something more involved than flipping the TOS bit — as in, if they've already connected their store to Jetpack, so what are they connecting it to now?

Reply:
This is the same verbiage as introduced in https://github.com/Automattic/woocommerce-services/pull/1177.

In this scenario, the user is in effect connecting their store to WooCommerce Services (SaaS layer). Since they're now opting into WCS, more of the Jetpack TOS apply to them, so we redisplay those terms.

Reply:
As https://github.com/Automattic/woocommerce-services/issues/1188 (being fixed) says,
> This banner directs users to connect to Jetpack.

Even if it doesn't say "Connect to Jetpack" in this case, why overload the term? I understand that we want to redisplay the terms, but in the event we do want the term "Connect" for accepting WCS terms, we should at least clarify that they're connecting to not-Jetpack in my opinion. cc @v18

Reply:
@dechov @v18 can we agree to discuss the wording separate from reviewing this bug fix PR?

Reply:
Yes! Sorry, I had missed the context of there being an actual bug separate from the verbal communication. Disregard ^^

Reply:
Agree with verbiage questions - opened separate issue for it here: https://github.com/Automattic/woocommerce-services/issues/1192#issue-265428329

Reply:
Thanks — and 👍  @jeffstieler for distinguishing urgency & prioritization of different matters